### PR TITLE
Delivery from trigger

### PR DESCRIPTION
**Pull request by tcezard**
On the back of #54 which addressed the project report this PR allow the delivery script to use the same trigger step to provide authorisation of release

### CODE REVIEW

**Comment by tcezard** [comment]
Integration tests are passing: see 2018-04-11_13\:22\:33.log
screen shot 2018-04-11 at 14 51 47

**Comment by tcezard** [comment]
Integration test are passing: See 2018-04-11_14:00:30.log
screen shot 2018-04-11 at 16 38 32

File: bin/deliver_reviewed_data.py [hunk 1]
```diff
@@ -55,57 +58,72 @@ def _now():
...
-            processes.sort(
-                key=lambda x: datetime.datetime.strptime(x.get('_created', '01_01_1970_00:00:00'),
-                                                         '%d_%m_%Y_%H:%M:%S'))
-            if not processes[-1].get('status', 'new') == 'finished':
-                raise EGCGError("Reviewed sample %s not marked as finished" % sample.get('sample_id'))
-            if sample.get('delivered', 'no') == 'no':
-                project_to_samples[sample.get('project_id')].append(sample)
-                self.all_samples_values.append(sample)
+            project_to_samples[sample.get('data').get('project_id')].append(sample.get('data'))
+            self.all_samples_dict[sample.get('data').get('sample_id')] = sample
```
Comment (mwhamgenomics) [inline bin/deliver_reviewed_data.py]: `.get` chains can be removed

**Comment by tcezard** [comment]
Updated integration tests.
screen shot 2018-04-23 at 10 09 58

File: bin/deliver_reviewed_data.py [hunk 2]
```diff
@@ -47,6 +47,8 @@
     os.path.dirname(os.path.dirname(os.path.abspath(__file__))), 'etc', 'delivery_email_template.html'
 )
 
+Release_LIMS_ste_name = 'Data Release Trigger EG 1.0 ST'
+
```
Comment (mwhamgenomics) [inline bin/deliver_reviewed_data.py]: What's a ste?

**Pull request merged**